src/vm/versionresilienthashcode.cpp

```diff
@@ -29,42 +29,53 @@ bool GetVersionResilientTypeHashCode(IMDInternalImport *pMDImport, mdExportedTyp
     bool hasTypeToken = true;
```

Comment:
This algorithm is part of the R2R binary contract. Changing it would require bumping the major R2R file format version.

Reply:
Should the version be changed from 2.2 to 3.0?

Reply:
We bump the file format major version once there is a strong reason to do it. I do not think a unification of the hash algorithm with CoreRT is a strong reason for the version bump.